Week-5/exercise_5_4/__tests__/calculator.test.js

```diff
@@ -0,0 +1,45 @@
+const mathOperations = require('../mathOperations');
+
+describe('Math operations', () => {
```

Comment:
The test cases should be in BDD format of given when then, u can use cucumber-test npm package

Reply:
I have already installed jest for testing, hope that work, also updating it to BDD format